react/features/authentication/components/native/WaitForOwnerDialog.tsx

```diff
@@ -106,11 +107,12 @@ class WaitForOwnerDialog extends Component<IProps> {
  */
 function mapStateToProps(state: IReduxState) {
     const { membersOnly, lobbyWaitingForHost } = state['features/base/conference'];
-    const serverURL = getServerURL(state);
+    const { conference } = getConferenceState(state);
+    const locationURL = conference?.getConnection()[JITSI_CONNECTION_URL_KEY];
```

Comment:
Why don't you take it from sthe state? We store it in base/connection. Also, it's a URL, so you should compare the hostname part.